crypto/rand/drbg_lib.c

```diff
@@ -75,8 +75,14 @@ typedef struct drbg_nonce_global_st {
 } DRBG_NONCE_GLOBAL;
 
 /* NIST SP 800-90A DRBG recommends the use of a personalization string. */
+#ifdef CHARSET_EBCDIC
+/* NOTE: If you change DRBG_DEFAULT_PERS_STRING, you will need to change this */
```

Comment:
IMO it would be better to make this change in rand_lcl.h, i.e.

    #ifdef CHARSET_EBCDIC
    # define DRBG_DEFAULT_PERS_STRING { 0x4f .... }
    #else
    # define DRBG_DEFAULT_PERS_STRING "OpenSSL NIST SP 800-90A DRBG"
    #endif

Reply:
Moved as you suggested